src/Infrastructure/Persistence/Doctrine/Mapping/Regulation.RegulationOrder.orm.xml

```diff
@@ -3,6 +3,9 @@
   <entity
     name="App\Domain\Regulation\RegulationOrder"
     table="regulation_order">
+    <indexes>
+        <index columns="end_date"/>
```

Comment:
Où serait utilisé cet index ? Je n'ai pas connaissance d'une query où on va filtrer par endDate ?

Reply:
OK vu, c'est pour le `$isPermanent` qui ajoute une clause endDate IS NULL / IS NOT NULL

C'est peut-être un cas similaire à un filtre sur le genre : faible sélectivité qui fait que l'index pourrait ne pas être utilisé en pratique

Reply:
Dans `RegulationOrderRecordRepository::findRegulationsByOrganizations`